src/openapi/streaming/streaming.ts

```diff
@@ -417,6 +426,13 @@ class Streaming extends MicroEmitter<EmittedEvents> {
                 break;
 
             case this.CONNECTION_STATE_RECONNECTING:
+                log.info(
+                    LOG_AREA,
+                    'Connection disconnected. Reconnecting...',

```

Comment:
This wording reflects that connections don't get reconnected, you just make a new connection. Reconnecting is just something the Connection class here does